linux_os/guide/system/network/network-firewalld/ruleset_modifications/configure_firewalld_rate_limiting/rule.yml

```diff
@@ -30,7 +30,7 @@ identifiers:
 
 references:
     disa: "2385"
-    nist: SC-5
+    nist: SC-5,SC-7(21)
```

Comment:
For rate-limiting, would any of the SC-5 controls be more applicable?

https://nvd.nist.gov/800-53/Rev4/control/SC-5